### PR TITLE
Add query fetch and indexing metrics per index mode

### PR DESCRIPTION
**Pull request by dnhatn**
This change exposes query, fetch and indexing metrics for each index mode.

### CODE REVIEW

File: server/src/main/java/org/elasticsearch/monitor/metrics/IndicesMetrics.java [hunk 1]
```diff
@@ -72,13 +73,48 @@ private static List<AutoCloseable> registerAsyncMetrics(MeterRegistry registry,
             );
             metrics.add(
                 registry.registerLongGauge(
-                    "es.indices." + name + ".bytes.total",
+                    "es.indices." + name + ".size",
```
Comment (dnhatn) [inline server/src/main/java/org/elasticsearch/monitor/metrics/IndicesMetrics.java:76]: I've renamed this metrics.

**Comment by dnhatn** [comment]
run elasticsearch-ci/part-1

**Comment by dnhatn** [comment]
@martijnvg This is ready for review.

**Review by martijnvg — APPROVED** [review]
LGTM

**Comment by dnhatn** [comment]
Thanks Martijn!

**Pull request merged**